modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2163,6 +2164,12 @@
 		end
 	end
 
+	function m.supportOpenMP(cfg)
+		if cfg.openmp == "On" then
+			m.element("OpenMPSupport", nil, "true")
+		end
```

Comment:
Let's output false here if "openmp" is set to false.  This way, we aren't relying on the exporter's default.